src/monitoring_service/cli.py

```diff
@@ -53,6 +56,30 @@ def setup_logging(log_level: str) -> None:
     )
 
 
+def check_gas_reserve(web3: Web3, private_key: str) -> bool:
+    """ Check periodically for gas reserve in the account """
+    gas_price = web3.eth.gasPrice
+    gas_limit = GAS_REQUIRED_FOR_MS_MONITOR
+    estimated_required_balance = gas_limit * gas_price * 10
```

Comment:
Can wyou make the 10 a variable?

Reply:
done, added it to the constants as `DEFAULT_GAS_BUFFER_FACTOR`